lib/src/test/java/io/ably/lib/test/realtime/RealtimePresenceTest.java

```diff
@@ -1949,81 +1943,131 @@ public void realtime_presence_get_throws_when_channel_failed() throws AblyExcept
 	 * Test if after reattach when returning from suspended mode client re-enters the channel with the same data
 	 * @throws AblyException
 	 *
-	 * Tests RTP17
+	 * Tests RTP17, RTP19, RTP19a, RTP5f
```

Comment:
It tests automatic re-enter feature described in RTP17. Test is done by injecting a member into internal presence map by tricking the client library that enter was confirmed by the server (though it really didn't) and then suspending and resuming the connection. Then server is queried using REST if there was a re-enter